app/app.py

```diff
@@ -16,20 +17,26 @@ def config_page():
 def data_summary_page():
     watchlist = request.form.getlist("itemDropdown")
     # print(watchlist)
-    # get_prices(watchlist)
-    return render_template("data_summary.html", options=watchlist)
+    prices_dict = get_prices(watchlist)
+    return render_template("data_summary.html", prices_dict = prices_dict)
 
-#def get_prices(list):
+def get_prices(list):
     api_codes = {
         "Corn": "TFGRAIN/CORN",
+        "Oats": "CHRIS/CME_O1",
+        "Wheat": "CHRIS/CME_W1",
+        "Soybean": "CHRIS/CME_S1",
         "Dairy": "CHRIS/CME_DA1",
     }
+    prices_dict = {}
     for item in list:
         url = "https://data.nasdaq.com/api/v3/datasets/" + api_codes[item] + "?api_key=7PvJ27fqD-s1fgvKWZgk"
         response =  urllib.urlopen(url)
```

Comment:
This threw me an error -- looks like this method has been deprecated for Python 3: https://stackoverflow.com/questions/3969726/attributeerror-module-object-has-no-attribute-urlopen.

Maybe try the solution that they suggest in that article and also double check that you're running Python 3 on your system, just so we're staying consistent.